### ISSUE TITLE
Add Handbrake support

### ISSUE THREAD

**Issue by GamerBene19** [revenz/FileFlows#33]
It would be really cool if this tool would support Handbrake.
I do have a preset that I would like to apply to some videos and I think this tool would be a good fit.

**Comment by revenz** [comment]
Since handbrake just uses FFMPEG under the hood, there is no reason or incentive to add support for handbrake.   It's not as simple as "just adding handbrakecli", all plugins/nodes would then have to have two execution paths, different parameters, different output handling etc.

You should be able to break down your handbrake profile into more workable chunks with the FFMPEG Builder nodes.

**Comment by GamerBene19** [comment]
> You should be able to break down your handbrake profile into more workable chunks with the FFMPEG Builder nodes.

Yes that's exactly what I did do yesterday evening. I actually was about to close this issue aswell.

The one thing that I was wondering - and which perhaps could be made more clear - is if FileFlows is able to copy audio/subtitle streams (as that was the main "feature" i was missing from my profile that I did not find in building blocks).
The answer (which I got from [here](https://youtu.be/G6r3LksBE5g?t=925)) is yes everything that is not touched is copied over by default.
